### ISSUE TITLE
Monkey patcher does not work well with global namespace

### ISSUE THREAD

**Issue by gwigz** [kahlan/kahlan#235]
So I've got something like this:

```php
namespace {
    use Utilities\MyOtherClass

    class MyClass
    {
        public function __construct()
        {
                $this->test = new MyOtherClass;
        }
    }
}
```

When we use it with Kahlan, `new MyOtherClass` is patched `Monkey::patched(__NAMESPACE__ , 'OtherMethod', false...`, which gives an error like this:

```
Fatal error: Class 'MyOtherClass' not found
```

I'm __guessing__ it should be patched something along the lines of `Monkey::patched(__NAMESPACE__ , 'Utilities\MyOtherClass', false...` instead?

**Comment by jails** [comment]
Good catch :+1: , yeah looks like there was an issue with the namespace curly brace syntax.
Should be fixed in the (dev branch)[https://github.com/kahlan/kahlan/commit/3a813645bc80c1e5bfe010e90bf725a389ec673f].

If it's ok I'll create a new release.

**Comment by gwigz** [comment]
Yup, that's resolved the issue.

Thanks! 👍

**Comment by jails** [comment]
Awesome, it's now merged in `3.1.12`
